### ISSUE TITLE
I made one!

### ISSUE THREAD

**Issue by Minorplanets76** [geezacoleman/owl-driver-board#1]
![IMG_1451](https://github.com/geezacoleman/owl-driver-board/assets/107326485/70af16e2-acb5-4977-9a2c-948e31928d47)
I built one by hand (dumb idea), but it seems to be working OK except for the LED's, i.e. the Pi has been running fine for extended periods, and the outputs can be switched on and off.  I don't think my leds are wrong, but I'm veryu much an amatuer.

An issue I see is that the camera cable is hard up against the bottom of the board and it running over fuses and power and stuff so might not be reliable long term in a dusty,vibrating environment. It might be better to have a penetration in the board.

![IMG_1449](https://github.com/geezacoleman/owl-driver-board/assets/107326485/366d601b-e986-434e-8334-5c104cc1169a)
There's a similar issue where one of the outputs is a bit hidden behind the USB3 ports.  Probably not an issue unless using ferrules.

Anyway, I'll continue to test.  I'm considering doing a less featured board with smd components on one side only...might be a little cheaper and easier.

Thanks for the design!

**Comment by geezacoleman** [comment]
This is very exciting to see! How did you solder/assemble it?

Thanks for the feedback too - having a slot for the cable is probably a good idea. I was wondering why it wasn't so compact on ours, but I think in the manufactured one it used longer standoffs for the GPIO, which provided more room between the Pi and the base of the top board. This also meant the USB port wasn't blocking the outputs. Still, the cable does sit up against the base of the board - perhaps at least some rubber tape could increase longevity.

![20220930_203652](https://github.com/geezacoleman/owl-driver-board/assets/51358498/dcab3996-1478-45ba-8228-61bfa2a6b4af)

Which features would you retain/remove? We could do a couple of versions depending on what people are after.

**Comment by Minorplanets76** [comment]
I hand soldered it, which probably wasn't a good idea.  JLPCB didn't have the wago connectors and I didn't really want 10, plus 2 sided assembly seemed fairly pricey so I figured why not.  It wasn't too bad, but I'm not sure if I'm doing the fuses right.Takes a long time so more out of interest than anything. Tape is a good idea, I'll do that.

I've spent a couple of hours on another version to try and get smd parts on one side.  Haven't managed to get it to fit yet.  I took out the PWM chip and various connectors so that it is just the power supply for the Pi and the Mosfet outputs.  Basically copied the schematic, and took out stuff that didn't seem essential.  I'm assuming PWM could be done in software, but have no idea if that would be true with the OWL overhead.  Using the same part numbers.  Probaby going have to have wires solder to the board.
[Schematic_Owl Driver Board_2023-07-21.pdf](https://github.com/geezacoleman/owl-driver-board/files/12114677/Schematic_Owl.Driver.Board_2023-07-21.pdf)
The only extra I have pondered is to hook up the POE pins assuming battery volatge used on a dumb POE injector.  Might be useful if you go wired network vs wireless.

**Comment by blinken** [comment]
Hey @Minorplanets76 , just to let you know we've completed some small revisions (including adding a camera slot!). We've also switched to decent screw terminals rather than the wago connectors as they were a bit of a pain to use.

This is available as [release v2.1 ](https://github.com/geezacoleman/owl-driver-board/releases/tag/v2.1) and we'll be doing a small production run of 10x.

I'm going to go ahead and close this issue, but I really appreciate the feedback. If you still need a board, shoot me a message!

![signal-2024-04-07-202754_003](https://github.com/geezacoleman/owl-driver-board/assets/250444/4caaeda9-61d3-4a44-978f-8cb40f597840)
